src/Dev/Upgrade/UpgradeBootstrap.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace SilverStripe\Dev\Upgrade;
+
+use BadMethodCallException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Provides upgrade interface for bootstrapping.
+ *
+ * Note: This class is intended to be loaded from outside of a SilverStripe application
+ * and should not reference any SilverStripe API.
+ *
+ * See https://github.com/silverstripe/silverstripe-upgrader/ for information
+ * on running this task.
+ */
+class UpgradeBootstrap
+{
+    /**
+     * List of files to install.
+     * Set to true if the file should be re-installed if it doesn't exist.
+     *
+     * @var array
+     */
+    protected $files = [
+        '.htaccess' => true,
+        'index.php' => true,
+        'install.php' => false,
+    ];
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @param $basePath
+     */
+    public function __invoke(InputInterface $input, OutputInterface $output, $basePath)
+    {
+        // Fail if destination isn't writable
+        $this->ensureWritable($basePath);
+
+        // Check source
+        $source = $basePath . '/vendor/silverstripe/recipe-core';
+        if (!is_dir($source)) {
+            throw new BadMethodCallException("silverstripe/recipe-core is not installed.");
+        }
+
+        // Copy scaffolded files from recipe-core
+        $output->writeln("Upgrading project bootstrapping files:");
+        foreach ($this->files as $file => $canCreate) {
+            $fileSource = $source . '/' . $file;
+            $fileDest = $basePath . '/' . $file;
+
+            // Skip if we should only upgrade existing files
+            if (!$canCreate && !file_exists($fileDest)) {
+                continue;
+            }
+            $output->writeln("  - Upgrading <info>{$file}</info>");
```

Comment:
This will revert customisations in an existing `.htaccess`file, such as static publisher routing, or denying access to additional files, right? I guess for all these changes performed by the importer, we expect users to manually review them before committing. But I've added clearer instructions around this anyway: https://github.com/silverstripe/silverstripe-framework/pull/7463